### DISCUSSION TITLE
v REPL should support `-cc` option

### DISCUSSION THREAD

**Discussion by ntrel** [vlang/v#14650] [Ideas]
```
C:\git\v>v -cc gcc
v : unknown command
Run  v help  for usage.
```
I want to do this because `tcc` doesn't always work on my system:
```
>>> mut a := [1,2]
==================
C:/Users/ntrel/AppData/Local/Temp/v_0/.noprefix.01G0H5QQQV7F4HEWT2FPQ64A0D.vrepl_temp.6468658124331114913.tmp.c:18115: warning: assignment from incompatible pointer type
tcc: error: undefined symbol 'CreateHardLinkW'
```

**Comment by JalonSolov** [comment]
As a workaround, try `VFLAGS='-cc gcc' v`.

**Comment by vincenzopalazzo** [comment]
Well, the workaround is the best solution, also because gcc is slower than tcc, the reaction time of the reply is worst with GCC.

BTW https://github.com/vlang/v/compare/master...vincenzopalazzo:macros/repl_cc?expand=1

**Comment by spytheman** [comment]
I do agree that setting the option on the command line should work. A workaround, or hardcoding the compiler in the repl is less optimal.
